### DISCUSSION TITLE
Платформа управленческого учета и отчетности для группы компаний

### DISCUSSION THREAD

**Discussion by a9341** [totumonline/totum-ru-issues-and-discussions#132] [Сделанные проекты]
# Задача
Необходима единая централизованная система управленческой отчетности для группы компаний. До начала проекта управленческая отчетность велась в надстройках 1С и в Excel с периодичностью подготовки 1 раз в месяц.
- Группа компаний осуществляющих разную деятельность: торговля, оказание услуг, производство.
- Первичный учет ведется в 1С, большой объем проводок: 100 тыс. проводок в месяц.
- Необходимо привести разрозненные данные к единому формату чтобы обеспечить управленческий учет и отчетность по проектам, по компаниям, по периодам
- Возможность из общего отчета провалиться в первичный документ
- Желательно обновлять данные отчетности не реже 1 раза в сутки

# Результат
На базе Тотум был создан бэкенд и фронтенд системы учета и отчетности для Клиента. Настроена автоматическая выгрузка "сырых" данных из 1С и передача в Тотум. В Тотуме данные из разных баз 1С приводятся в единый формат. Из подготовленных данных внутри Тотума собираются отчеты с которыми работают конечные пользователи - менеджмент и финансовый отдел.

**Comment by gorbarov** [comment]
Скринами бы поделились, данные можно замазать, а вот внешний вид таблиц, общая архитектура и тд для понимающий будет понятна и, вероятно, полезна ;-)
